app/lib/date_parser.rb

```diff
@@ -0,0 +1,18 @@
+class DateParser
+  def self.parse(date)
+    if date_regex.match? date
+      DateTime.strptime(date, "%m/%d/%y")
+    else
+      nil
+    end
+  rescue ArgumentError
+    nil
```

Comment:
Do we need these `nil`s? 
 Would this suffice:
```
def self.parse(date)
  DateTime.strptime(date, "%m/%d/%y") if date_regex.match? date
  rescue ArgumentError
end
```

Reply:
Yeah I like that a lot more, a lot less verbose since we're just returning nil in all other cases anyways.